src/internal/kopia/wrapper_test.go

```diff
@@ -765,18 +766,24 @@ func (suite *KopiaIntegrationSuite) TearDownTest() {
 }
 
 func (suite *KopiaIntegrationSuite) TestBackupCollections() {
+	c1 := exchMock.NewCollection(
+		suite.storePath1,
+		suite.locPath1,
+		5)
+	// Add a 4k chunk of data that should be compressible. This helps check
+	// compression is enabled because we do some testing on the number of bytes
+	// uploaded during the first backup.
+	c1.Data[0] = []byte(strings.Repeat("abcdefgh", 512))
```

Comment:
Just for my own understanding, where exactly are checking for compression?

Reply:
good question! Unfortunately we can't do a direct check on the uploaded data blobs to check for compression. Instead, what we do is check the number of bytes uploaded ([here](https://github.com/alcionai/corso/blob/3e43028a883bb2a38b3fe96eaeab745f31193445/src/internal/kopia/wrapper_test.go#L899) and the assertion below). From manual testing, these assertions pass when compression is enabled but fail if it's not